release/release_tests.yaml

```diff
@@ -2277,7 +2277,7 @@
 # Serve tests
 ########################
 
-- name: serve_single_deployment_1k_noop_replica
+- name: serve_scalability_1k_replicas
```

Comment:
let's drop "1k replicas" from the name, filename, etc. because we will presumably be increasing the number over time :)